src/main/AndroidManifest.xml

```diff
@@ -13,11 +12,14 @@
         android:icon="@drawable/launcher_icon"
         android:label="@string/app_name"
         android:theme="@style/AppTheme">
-        <activity android:name=".AudioRecordActivity">
+        <activity
+            android:name=".BluetoothRecordActivity"
+            android:exported="true">
```

Comment:
Please revert. Audio recording, which is demonstrated by `AudioRecordActivity`, is the easier case. Using Bluetooth is only for people who want it.